pkg/route/api/v1/conversion.go

```diff
@@ -10,8 +10,16 @@ import (
 func addConversionFuncs(scheme *runtime.Scheme) {
 	err := scheme.AddDefaultingFuncs(
 		func(obj *RouteSpec) {
-			if len(obj.To.Kind) == 0 {
-				obj.To.Kind = "Service"
+			if obj.AdditionalTos == nil {
+				obj.AdditionalTos = make([]RouteTargetReference, 0)
+			}
+		},
+		func(obj *RouteTargetReference) {
+			if len(obj.Kind) == 0 {
+				obj.Kind = "Service"
+			}
+			if obj.Weight == 0 {
+				obj.Weight = 1
```

Comment:
Lets default to 100, so users can easily alter relative weights.  Setting 1 means that someone wanting to provide a low weight B (a common case) would have to change both fields.